IoTuring/__init__.py

```diff
@@ -28,6 +28,7 @@ def loop():
 
     # add -c to configure with the menu
     if len(sys.argv) > 1 and sys.argv[1] == "-c":
+        configurator.configuratorIO.checkConfigurationFileInOldLocation()
```

Comment:
I think this should be the other way. This config migration is just a unique situation now, and in the future this won't be important. So I think first the new location should be checked, and if there is no config in thee new location, check the old.

New location should take precedent.